File: api/api/models/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace api.models
{
    public class Customer:User
    {
     

        public Adress BillingAdress { get; set; }
        public Adress DeliveryAdress { get; set; }
        public DateTime BirthDayDate { get; set; }

    }
}

File: front/src/app/models/Customer.ts
import { Adress} from "./adress";
import { User } from "./User";

export class Customer extends User{

    private _BirthdayDate: Date;
    private _DeliveryAdress: Adress; 

constructor(){
super()
}

public get BirthdayDate(): Date {
    return this._BirthdayDate;
}
public set BirthdayDate(value: Date) {
    this._BirthdayDate = value;
}
public get DeliveryAdress(): Adress {
    return this._DeliveryAdress;
}
public set DeliveryAdress(value: Adress) {
    this._DeliveryAdress = value;
}
}